dsalgo/string_algo.py

```diff
@@ -0,0 +1,156 @@
+class Strings:
+    def encode(self, strs):
+        """
+        Encodes a list of strings to a single string.
+        params: List[str]
+        returns: str
+        """
```

Comment:
Not able to understand clearly what your function is doing by this. Please elaborate